definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -2092,6 +2092,13 @@
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "16/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "SUFixCaseEvent",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
```

Comment:
We should probably give other users 'R' access for this... so they can see what has happened

Reply:
Marking as resolved as changes made